frontends/election-manager/src/screens/print_test_deck_screen.tsx

```diff
@@ -574,15 +575,35 @@ export function PrintTestDeckScreen(): JSX.Element {
         />
       )}
       <NavigationScreen>
-        <Prose>
+        <Prose maxWidth={false}>
           <h1>{pageTitle}</h1>
           <p>
-            Select desired precinct for <strong>{election.title}</strong>.
+            Print all or select a precinct to print the L&A Package for that
+            precinct.
           </p>
+          <HorizontalRule />
+          <p>Each Precinct L&A Package contains:</p>
+          <ol>
+            <li>
+              The Precinct Tally Report is the expected results from scanning
+              the marked ballots for this precinct.
```

Comment:
This being a full sentence reads a bit funny to me since we're also leading into it with "Each Precinct L&A Package contains".

I end up reading it as "Each Precinct L&A Package contains The Precinct Tally Report is the expected results from scanning the marked ballots for this precinct."

What do you think of changing this item to "The Precinct Tally Report. This is the expected result from scanning the marked ballots for this precinct." or something along those lines? This will also make this list item better match the other list items.

Reply:
Changing to Ginny's suggestion of "A Precinct Tally Report — the expected results of the precinct."